en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md

```diff
@@ -0,0 +1,352 @@
+
+## Overview
+
+Key airframe features:
+
+- Compact and easy to transport
+- Pre installed actuators
+- Quick release wing connecting system
+- Transport case in kit included
+- ~35 minute long flight times (depending on takeoff weight)
+- VTOL enables flying in locations where a fixed wing couldn't fly
+- Battery and charger included in the kit
+- Easy overall build
+- Space to mount FPV and/or action camera in the front
+
+Depending on the final takeoff weight the hover time might be limited (there is not a lot of air circulation inside the fuselage when the vehicle is hovering so the ESCs might overheat).
+
+## Where to Buy
+
+- [OMP-Hobby](https://www.omphobby.com/OMPHOBBY-ZMO-VTOL-FPV-Aircraft-With-DJI-Goggles-And-Remote-Controller-p3069854.html)
+- [GetFPV](https://www.getfpv.com/omphobby-zmo-z3-vtol-fpv-1200mm-arf-plane-kit-no-fpv-system.html)
+- [FoxtechFPV](https://www.foxtechfpv.com/zmo-pro-fpv-vtol.html)
+
+## Flight Controller
+
+The following options have been tested:
+
+- [Auterion Skynode evaluation kit](../companion_computer/auterion_skynode.md)
+- [Pixhawk 6C](../flight_controller/pixhawk6c.md) with [PM02 V3](../power_module/holybro_pm02.md)
+- [Pixhawk 6C mini](../flight_controller/pixhawk6c_mini.md) with [PM02 V3](../power_module/holybro_pm02.md)
+
+The approximate maximum size of the FC is: 50x110x22mm
+
+## Additional Accessories
+
+- [GPS F9P (included in Skynode eval. kit)](../gps_compass/rtk_gps_holybro_h-rtk-f9p.md)
+- [GPS M9N (cheaper alternative to F9P)](../gps_compass/rtk_gps_holybro_h-rtk-m8p.md)
+- [Airspeed sensor (included in Skynode eval. kit)](https://www.dualrc.com/parts/airspeed-sensor-sdp33) — recommended for improved safety and performance
+- [Airspeed sensor (cheaper alternative)](https://holybro.com/products/digital-air-speed-sensor?pr_prod_strat=use_description&pr_rec_id=236dfda00&pr_rec_pid=7150470561981&pr_ref_pid=7150472462525&pr_seq=uniform)
+- [Lidar Lightware lw20-c (included in Skynode eval. kit)](../sensor/sfxx_lidar.md) (Optional)
+- [Lidar Seeed Studio PSK-CM8JL65-CC5 (cheaper alternative)](https://www.seeedstudio.com/PSK-CM8JL65-CC5-Infrared-Distance-Measuring-Sensor-p-4028.html)
+- [5V BEC](http://www.mateksys.com/?portfolio=bec12s-pro)
+- [Radio (RC) System](../getting_started/rc_transmitter_receiver.md) of your preference
+- [Servo cable extension cable male 30cm 10 pcs](https://www.getfpv.com/male-to-male-servo-extension-cable-twisted-22awg-jr-style-5-pcs.html)
+- [USB-C extension cable](https://www.digitec.ch/en/s1/product/powerguard-usb-c-usb-c-025-m-usb-cables-22529949?dbq=1&gclid=Cj0KCQjw2cWgBhDYARIsALggUhrh-z-7DSU0wKfLBVa8filkXLQaxUpi7pC0ffQyRzLng8Ph01h2R1gaAp0mEALw_wcB&gclsrc=aw.ds)
+- [3M VHB tape](https://www.amazon.in/3M-VHB-Tape-4910-Length/dp/B00GTABM3Y)
+- [3D-Printed mounts](https://github.com/PX4/PX4-user_guide/raw/main/assets/airframes/vtol/omp_hobby_zmo_fpv/OMP-Hobby-ZMO-3D-Prints.zip)

```

Comment:
FYI only -note this changed to a raw link rather than linking to the asset locally as you had done. Vuepress can't handle a direct link for some reason.